test/testcmdlineparser.cpp

```diff
@@ -151,39 +151,39 @@ class TestCmdlineParser : public TestFixture {
 
 
     void nooptions() {
-        REDIRECT;
+        REDIRECT
```

Comment:
would it be possible to tweak this macro also?

Reply:
Will do.